build/Dockerfile

```diff
@@ -1,12 +1,24 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13 AS builder
+
+RUN mkdir /custom-domains-operator
```

Comment:
Change `/custom-domains-operator` to `/src` for consistency and to make it more generic in preparation for boilerplating.